src/join/composer.jl

```diff
@@ -559,6 +653,10 @@ change in future releases.
   in `on` columns; if equal to `:equal` then `missing` is allowed and missings are
   matched; if equal to `:notequal` then missings are dropped in `df1` and `df2`
   `on` columns; `isequal` is used for comparisons of rows for equality
+- `order` : if `nothing` (the default) the order of rows in the result is
```

Comment:
Maybe we should use another symbol instead of `nothing`, like `:undefined`?

Reply:
makes sense. fixed